inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/Initializer.kt

```diff
@@ -27,15 +23,16 @@ import java.util.UUID
 internal object Initializer {
 
     private const val TAG = "IAM_InitWorker"
-    private const val ID_KEY = "uuid_key"
+    internal const val ID_KEY = "uuid_key"
 
     /**
      * This method returns a string of Android Device ID. Note: In order to get device ID without Context,
      * use HostAppInfoRepo.
      */
     @SuppressLint("HardwareIds") // Suppress lint check of using device id.
-    private fun getDeviceId(context: Context): String =
-            Settings.Secure.getString(context.contentResolver, Settings.Secure.ANDROID_ID) ?: getUuid(context)
+    private fun getDeviceId(context: Context, sharedUtil: SharePreferencesUtil) =
```

Comment:
This is ok, since in kotlin allows inferred return types.
`Settings.Secure.getString(context.contentResolver, Settings.Secure.ANDROID_ID) ?: getUuid(context, sharedUtil)` returns a `String` type

Reply:
👍